src/easylink/pipeline_schema_constants/tests.py

```diff
@@ -0,0 +1,124 @@
+from easylink.graph_components import (
+    EdgeParams,
+    InputSlot,
+    InputSlotMapping,
+    OutputSlot,
+    OutputSlotMapping,
+)
+from easylink.step import HierarchicalStep, InputStep, LoopStep, OutputStep, Step
+from easylink.utilities.validation_utils import validate_input_file_dummy
+
+SINGLE_STEP_NODES = [
```

Comment:
SINGLE_STEP_NODES, _EDGES, and _SCHEMA_PARAMS were oved from old `pipeline_schema_constants/integration_test.py` (the file deleted above). They are identical except for the names.